package.json

```diff
@@ -64,6 +66,8 @@
     "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.2",
-    "ember-try": "0.0.6"
+    "ember-jsdoc": "^1.2.0",
```

Comment:
I'm not sure you have the latest changes from the `develop` branch merged in, as this repo is not used anymore.